v2/samples/network/v1api20220701/v1api20220701_applicationgateway.yaml

```diff
@@ -0,0 +1,148 @@
+apiVersion: network.azure.com/v1api20220701
+kind: ApplicationGateway
+metadata:
+  name: aso-sample-application-gateway
+  namespace: default
+spec:
+  location: westus2
+  owner:
+    name: aso-sample-rg
+  azureName: aso-sample-application-gateway
+  autoscaleConfiguration:
+    maxCapacity: 2
+    minCapacity: 1
+  sku:
+    name: WAF_v2
+    tier: WAF_v2
+  gatewayIPConfigurations:
+  - name: app-gw-ip-config-1
+    subnet:
+      reference:  
+        group: network.azure.com
+        kind: VirtualNetworksSubnet
+        name: aso-sample-subnet
+  frontendIPConfigurations:
+  - name: app-gw-fip-config-1
+    privateIPAddress: "10.0.0.10"
+    privateIPAllocationMethod: Static
+    subnet:
+      reference:  
+        group: network.azure.com
+        kind: VirtualNetworksSubnet
+        name: aso-sample-subnet
+  - name: app-gw-fip-config-2
+    publicIPAddress:
+      reference:  
+        group: network.azure.com
+        kind: PublicIPAddress
+        name: aso-sample-publicip
+  frontendPorts:
+  - name: app-gw-fe-port-1
+    port: 443
+  sslCertificates:
+  - name: app-gw-sslcert-1
+    data: |
+      MIIKEQIBAzCCCdcGCSqGSIb3DQEHAaCCCcgEggnEMIIJwDCCBHcGCSqGSIb3DQEH
```

Comment:
IMO we should redact this secret, it won't work for users? Or will it? Is this just the publicData or private data too?

Reply:
this is the tls certificate attached to the app gateway to enable tls on the listners, It contains the full chain tls certificates in base64